Example/GiniVision/AppDelegate.swift

```diff
@@ -38,6 +38,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
     
+    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
+        // If the current root view controller is not the ScreenAPIViewController, it won't do anything.
```

Comment:
Why does it have to be the `ScreenAPIViewController `?

Reply:
Because it is the GiniVisionDelegate, maybe It should be explained.